mitodl/ocw-hugo-themes
# added redirects

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes https://github.com/mitodl/ol-infrastructure/issues/922

#### What's this PR do?
Adds redirect

#### How should this be manually tested?
- Click the latest netlify preview
- Navigate to /fairuse
- Verify it redirects
